packages/lb-components/src/components/pointCloudView/components/TitleButton/index.tsx

```diff
@@ -4,15 +4,16 @@ import { getClassName } from '@/utils/dom';
 
 interface IProps {
   title: string | React.ReactElement | null | undefined;
-  onClick: () => void;
+  onClick?: () => void;
   style?: React.CSSProperties;
+  hiedZoom?: boolean; // Hide zoom function
```

Comment:
Error word. `hied`. And you can directly use `onClick ` to determine whether to show the expand.

Reply:
And you can rename it from `onClick` to `expandClick`